log-2018-05-22.php

```diff
@@ -0,0 +1,7 @@
+<?php defined('BASEPATH') || exit('No direct script access allowed'); ?>
```

Comment:
Don't think you meant to include this one...

Reply:
Nope - that should have been git ignored. Will fix